### PR TITLE
"Take the Course" Steps (#issue 1466)

### PR DESCRIPTION
**Pull request by ofrank23447**
Under "Take the Course" (Step4 BeLLApp > Take the Course section), I added simple steps to guide virtual interns to take the virtual intern course quiz.  
image

### CODE REVIEW

File: pages/vi/vi-bellapps.md [hunk 1]
```diff
@@ -51,12 +51,16 @@ First, click the "Update Available" button and it will reload your homepage with
...
 
-To get to Publications, click on "Manager" from the dashboard, then the publications button. Take the course, and the last question will ask you to specify any problems or improvements/suggestions that you have. Remember, people taking these courses out in remote areas of the world will probably run into the same problems if you do not raise them now and let the team know. This is a very important task, and your help is much appreciated. 
+To get to Publications, click on "Manager" from the dashboard, then the publications button. Take the course, and the last question will ask you to specify any problems or improvements/suggestions that you have. Remember, people taking these courses out in remote areas of the world will probably run into the same problems if you do not raise them now and let the team know. This is a very important task, and your help is much appreciated.  
+**STEPS:**   
+1.Click on Courses (next to Library).  
+2.There is a course called Virtual Interns.  
+3.You will see a list of steps that test your knowledge of the discussed concepts from virtual interns first-steps.
```
Comment (kylemathias) [inline pages/vi/vi-bellapps.md]: Try to use markdown elements to make these steps a little cleaner.

Also, try to reword step 2 and step 3. Try to make it more concise and to the point.

Comment (ofrank23447) [reply pages/vi/vi-bellapps.md]: Okay, I will do that and let you know soon.

**Comment by ofrank23447** [comment]
http://rawgit.com/ofrank23447/ofrank23447.github.io/issue1466/#!./pages/vi/vi-bellapps.md  
image

File: pages/vi/vi-bellapps.md [hunk 2]
```diff
@@ -51,12 +51,16 @@ First, click the "Update Available" button and it will reload your homepage with
 
 We developed a course for aspiring virtual interns so that we could test the software used to build courses and find bugs/things to improve. It is your job to take the course and find out what needs to be fixed/improved from the student standpoint.
 
-To get to Publications, click on "Manager" from the dashboard, then the publications button. Take the course, and the last question will ask you to specify any problems or improvements/suggestions that you have. Remember, people taking these courses out in remote areas of the world will probably run into the same problems if you do not raise them now and let the team know. This is a very important task, and your help is much appreciated. 
+To get to Publications, click on "Manager" from the dashboard, then the publications button. Take the course, and the last question will ask you to specify any problems or improvements/suggestions that you have. Remember, people taking these courses out in remote areas of the world will probably run into the same problems if you do not raise them now and let the team know. This is a very important task, and your help is much appreciated.   
+**STEPS:**
```
Comment (kylemathias) [inline pages/vi/vi-bellapps.md]: could use a empty line above "STEPS:"

Comment (ofrank23447) [reply pages/vi/vi-bellapps.md]: Should I put that and let you know again?

**Comment by ofrank23447** [comment]
I fixed the issue.  
http://rawgit.com/ofrank23447/ofrank23447.github.io/issue1466/#!./pages/vi/vi-bellapps.md  
image

**Comment by ofrank23447** [comment]
@xyb994 Thank you very much

**Pull request merged**